app/services/requirements/scheduled_datetime_checker.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+module Requirements
+  class ScheduledDatetimeChecker
+    attr_reader :day, :month, :year, :time
+
+    FUTURE_TIME_PERIOD = { months: 14 }.freeze
+
+    def initialize(params)
+      @day = is_numeric?(params[:day]) ? params[:day].to_i : params[:day]
```

Comment:
Do we need to do this anymore? We only use it in the context of a string, so it doesn't look like we need to convert it to an integer :-).

Reply:
Yep, totally missed that!